app/helpers/assignments_helper.rb

```diff
@@ -0,0 +1,5 @@
+module AssignmentsHelper
+  def full_name_and_email(user)
+    "#{user.full_name} (#{user.email})"
```

Comment:
How would we feel about plopping this in the model? It feels more like a representation of the data in the object to me than a manipulation of it, but weak opinion weakly held there.

Reply:
Hmmm, my thought process was that this was presentation logic relating only to how the user is represented in the view 🤔 but I see your point!

I'm just going to leave it for now, so we can get it merged